e2e-tests/src/npg/helpers.js

```diff
@@ -1,5 +1,5 @@
 export const selectKeyboardForm = async () => {
-  const selectFormXPath = '/html/body/div[1]/div/div[2]/div/div[2]/div[2]/div[1]/div/div/div[1]';
+  const selectFormXPath = '/html/body/div[1]/div/main/div/div[2]/div[2]/div[1]/div/div/div[1]';
```

Comment:
Did you consider to add an id and replace this XPath with selector ?

Reply:
Yes, i had a discussion about why we are using xpath with simone i, and resulted that those divs targeted in these tests are imported from libraries and have no id or class that we can use to anchor them.